indico/cli/i18n.py

```diff
@@ -392,6 +454,9 @@ def _indico_command(babel_cmd, python, javascript, react, locale, no_check):
         click.secho(f'Error running {babel_cmd} for indico - {err}', fg='red', bold=True, err=True)
     if babel_cmd == 'ExtractMessages':
         remove_empty_pot_files(INDICO_DIR, python=python, javascript=javascript, react=react)
+        merge_pot_files(TRANSLATIONS_DIR / 'merged.pot', MESSAGES_POT, MESSAGES_JS_POT, MESSAGES_REACT_POT)
```

Comment:
Would it makes sense to also remove all the other resources if we're just gonna use the merged POT?

Reply:
Would that imply running an extraction during compile or fetch time? If yes, I'm not sure that's a good idea:

1) it makes the whole thing slower
2) the codebase may have changed in the meantime, so the pot files are not identical to those used earlier and which are on transifex

Reply:
There's no issue keeping them around.

> the codebase may have changed in the meantime, so the pot files are not identical to those used earlier and which are on transifex

Wouldn't this be a good a thing though? When compiling the resources I only care about what's currently in the source code not what we had at some point in the past

Reply:
true... even though I would not want to leave new pot files around (they would not belong in the commit that updates translations) during an unrelated operation.

Reply:
For now let's keep it as discussed with @tomasr8 . No harm in having the separate pot files and we can always get rid of it if necessary.